queries.sql

```diff
@@ -17,3 +17,65 @@ SELECT * FROM animals WHERE name != 'Gabumon';
 
 SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3; 
 SELECT * FROM animals WHERE weight_kg Between 10.4 AND 17.3;
+
+BEGIN;
+
+    UPDATE animals
+    SET species = 'unspecified';
+
+    SELECT * FROM animals;
+
+ROLLBACK;
+
+SELECT * FROM animals;
+
+UPDATE animals SET species='digimon' WHERE name LIKE '%mon';
+
+UPDATE animals SET species='pokemon' WHERE species IS NULL;
+
+SELECT * FROM animals;
+
+BEGIN;
+
+    DELETE FROM animals;
+
+ROLLBACK;
+
+SELECT * FROM animals;
```

Comment:
- [ ] According to the requirements you should check if the table is deleted after the DELETE query.

Reply:
- Now, take a deep breath and... Inside a transaction delete all records in the animals table, then roll back the transaction.
- After the rollback verify if all records in the animals table still exists. After that, you can start breathing as usual ;)

These are the requirements for this part and I did the same. According to the requirements I need to check the table after rollback which is what I am doing.